sdk/identity/azure-identity/azure/identity/_credentials/vscode_credential.py

```diff
@@ -45,9 +47,8 @@ def get_token(self, *scopes, **kwargs):
 
         refresh_token = get_credentials()
         if not refresh_token:
-            raise CredentialUnavailableError(
-                message="No Azure user is logged in to Visual Studio Code."
-            )
-        token = self._client.get_cached_access_token(scopes) \
-                or self._client.obtain_token_by_refresh_token(refresh_token, scopes, **kwargs)
+            raise CredentialUnavailableError(message="No Azure user is logged in to Visual Studio Code.")
+        token = self._client.get_cached_access_token(scopes) or self._client.obtain_token_by_refresh_token(
```

Comment:
when calling `obtain_token_by_refresh_token` should we cache the new refresh token?

Reply:
Sorry, you mean cache refresh token or access token? access token is already cached in get_cached_access_token